parts/k8s/kubernetescustomscript.sh

```diff
@@ -496,6 +496,25 @@ configAddons() {
     fi
 }
 
+installAzCli(){
+    AZ_REPO=$(lsb_release -cs)
+    echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ $AZ_REPO main" | \
+    sudo tee /etc/apt/sources.list.d/azure-cli.list
+    retrycmd_if_failure 3 1 30 curl -L https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
+    apt_get_update && apt_get_install 2 20 120 azure-cli
```

Comment:
Any reason why only 2 retries?

Reply:
The thought here was: we don't want to delay/block deployment for logs so being resilient in this step doesn't matter a lot

Reply:
Let's reduce timeout to 60 and retry 3 times